README.md

```diff
@@ -42,7 +42,16 @@ For more information and distro-specific instructions, see the [Linux installati
 
 ### Windows
 
-`gh` is available via [scoop][], [Chocolatey][], and as downloadable MSI.
+`gh` is available via [WinGet][], [scoop][], [Chocolatey][], and as downloadable MSI.
+
+
+#### WinGet
+
+Install:
+
+```powershell
+winget install GitHub.cli
```

Comment:
1. It seems like this package is aliased as `gh`. Could the installation instructions simply refer to it as `gh`, for brevity?
2. What would be the upgrade instructions? Do you think you could create a table with install/upgrade columns like in the other sections of this document?

Reply:
Ok